pod/perlfunc.pod

```diff
@@ -6164,8 +6182,7 @@ effect as
         $ARRAY[++$#ARRAY] = $value;
```

Comment:
Shouldn't this be `$ARRAY[@ARRAY] = $value;`?

Reply:
I think the intent is to show (to someone intimately familiar with how `$#prefix` works) that the operation increases the size of the array and then assigns the element there. But I agree with @haarg  that this entire example doesn't give any useful information.

Reply:
I'm good with removing, but if we don't we should restate as I diid. The original is cryptic, and unnecessary (as you know there is no need to presize the array to store into an index) and we shouldn't explain one concept by invoking a separate and even more complex concept. Assigning a value to the next available index is often done the way I wrote it (ive done it myself  many times [although usually in two statements] and ive seen others do it as well), but I can't recall ever having seen it done the way it's shown in that example.